ecosystem/sep-0031.md

```diff
@@ -857,9 +802,18 @@ In certain cases the Receiving Anchor might need to request updated information
 the bank tells the Receiving Anchor that the provided Receiving Client's name is incorrect or missing a middle initial.
 Since this information was sent via SEP-12, the transaction should go into the `pending_customer_info_update` state
 until the Sending Anchor makes another `SEP-12 PUT /customer` request to update. The Sending Anchor can check which
-fields need to be updated by making a `SEP-12 GET /customer` request including the `id` or `account` & `memo`
-parameters. The Receiving Anchor should respond with a `NEEDS_INFO` status and `last_name` included in the fields
-described.
+fields need to be updated by making a `SEP-12 GET /customer` request including either the customer id `id`, or a
+combination of SEP-31 transaction id `transaction_id` and customer type `type`, which can only be `sep31-sender` or
```

Comment:
I think we can recommend this way of collecting KYC info, but it doesn't have to be the _only_ way.